WingIt/LoginViewController.swift

```diff
@@ -184,12 +192,11 @@ class LoginViewController: UIViewController, UIWebViewDelegate, UITextFieldDeleg
             if (header == "" && !once) {
                 errorLabel.text = "Enter your eVision details"
                 displayLoginFields()
-                if PWIsStored{
+                if PWIsStored {
                     usernameField.text = retrieveStoredUsername()
-                    usernameField.backgroundColor = UIColor.yellow
```

Comment:
Rename this autoFillColor

Reply:
If you want to be pedantic about variable names then we should have been from the start, but fair enough haha.

Reply:
This is fixed and also the above.
Was done from linux however so should make sure to wait for Travis to test the build.